src/test/java/com/exasol/adapter/dialects/elasticsearch/ElasticSearchSqlDialectIT.java

```diff
@@ -264,6 +269,14 @@ void testDocumentWithNestedProperty() throws IOException {
                 table().row("str", "inner_str").matches(TypeMatchMode.NO_JAVA_TYPE_CHECK));
     }
 
+    @Test
+    void testSelectAllColumns() throws IOException {
+        this.indexDocument(createObjectBuilder().add("c1", "str").add("c2", 42).add("c3", 3.14).build());
+        final String query = "SELECT * FROM " + getVirtualTableName();
+        assertVirtualTableContentsByQuery(query,
+                table().row("str", "str", 42, 3.140000104904175).matches(TypeMatchMode.NO_JAVA_TYPE_CHECK));
+    }
```

Comment:
There was a note in the user guide saying that `SELECT *` did not work, so I added a test.

Reply:
:+1: